packages/curve-ui-kit/src/widgets/Disclaimer/Disclaimer.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState, useEffect } from 'react'
+import { useNavigate, useLocation } from 'react-router-dom'
+import { t } from '@lingui/macro'
+
+import Stack from '@mui/material/Stack'
+
+import { TabsSwitcher } from 'curve-ui-kit/src/shared/ui/TabsSwitcher'
+import { SizesAndSpaces } from 'curve-ui-kit/src/themes/design/1_sizes_spaces'
+
+import { LastUpdated } from './LastUpdated'
+import { Footer } from './Footer'
+import { TabPanel } from './TabPanel'
+import { Dex } from './Tabs/Dex'
+import { LlamaLend } from './Tabs/LlamaLend'
+import { CrvUsd } from './Tabs/CrvUsd'
+import { SCrvUsd } from './Tabs/SCrvUsd'
+
+const { MaxWidth } = SizesAndSpaces
+
+const tabs = [
+  { id: 'dex', label: t`Dex` },
+  { id: 'lend', label: t`LlamaLend` },
+  { id: 'crvusd', label: t`crvUSD` },
+  { id: 'scrvusd', label: t`Savings crvUSD` },
+] as const
+type TabId = (typeof tabs)[number]['id']
+const DEFAULT_TAB: TabId = 'dex'
+
+type Props = {
+  className?: string
+}
+
+export const Disclaimer = ({ className }: Props) => {
+  const navigate = useNavigate()
+  const location = useLocation()
+
+  const getTabFromUrl = () => {
```

Comment:
This should be  moved to the global scope to avoid the eslint warning (`useEffect has a missing dependency: 'getTabFromUrl'`)

Reply:
Should we configure lint warnings as errors? Missed this one as it's hidden away as a small step in `yarn lint` on GitHub Actions.

Reply:
Moved `getTabFromUrl` outside of the function component. This amde me realize `handleTabChange` is dependent on both `navigate` and `location` and is passed as a callback to `TabSwitcher` so I've wrapped it in a `useCallback` as well.